components/director/internal/metrics/collector.go

```diff
@@ -24,6 +25,7 @@ type Collector struct {
 	hydraRequestTotal      *prometheus.CounterVec
 	hydraRequestDuration   *prometheus.HistogramVec
 	clientTotal            *prometheus.CounterVec
+	mutationCount          *prometheus.CounterVec
```

Comment:
We are using this counter for both graphql mutations and queries so `gqlOperationCount` may be mora appropriate name